spec/models/endpoint_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Endpoint, type: :model do
+  context 'when creating an endpoint' do
+    subject { described_class.new }
+
+    it 'requires a kimono_id' do
```

Comment:
Up to you, but you could also change this to `errors without required fields`.  Right now that field happens to be `kimono_id`, but later on, it could be that plus X more fields.